Src/BootCamp.Chapter/ContactsCenter.cs

```diff
@@ -1,16 +1,35 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.IO;
+using System.Linq;
 using System.Text;
 
 namespace BootCamp.Chapter
 {
     public class ContactsCenter
     {
-        private readonly List<Person> _people;
+        private readonly List<Person> _people = new List<Person>();
 
         public ContactsCenter(string peopleFile)
         {
-            // load people
+            string[] people;
+
+            try
```

Comment:
Ok. If I understand this correctly I need to catch two exceptions (File and DirectoryNotFound) which both throws FileNotFound (this because DirectoryNotFound expects FileNotFound in the tests). 

I throw a general exception when file is empty (also expected in the tests).